client/app/vps/backup-storage/order/vps-order-backup-storage.html

```diff
@@ -0,0 +1,44 @@
+<section class="container-fluid cui-page__content cui-page__content-flex">
+
+    <div class="row d-lg-flex">
+        <div class="col-xm-12">
+            <p>OPT:{{$ctrl.model.optionDetails}}</p>
```

Comment:
hardcoded OPT?  Will this be meaningful for every language?

Reply:
I cleary forget about this, I will remove it